pandas/core/series.py

```diff
@@ -2769,6 +2769,11 @@ def corr(
         float
             Correlation with other.
 
+        .. note::
+
+            Automatic data alignment: as with all pandas operations, automatic data alignment is performed for this method.
+            ``corr`` automatically considers values with matching indices.
```

Comment:
maybe just add this to the "notes" section below?

Reply:
okay, sure